vhdbuilder/packer/install-dependencies.sh

```diff
@@ -289,6 +289,15 @@ else
     retagContainerImage "docker" ${watcherFullImg} ${watcherStaticImg}
 fi
 
+# doing this at vhd allows CSE to be faster with just mv
+cniUnTarNClean() {
+  CNI_TGZ_TMP=${VNET_CNI_PLUGINS_URL##*/}
+  CNI_DIR_TMP=${CNI_TGZ_TMP%.tgz}
+  mkdir "$CNI_DOWNLOADS_DIR/${CNI_DIR_TMP}" 
+  tar -xzf "$CNI_DOWNLOADS_DIR/${CNI_TGZ_TMP}" -C $CNI_DOWNLOADS_DIR/$CNI_DIR_TMP
```

Comment:
can we add a check to run-test that the binaries exist where cse thinks they do?

if no, can you clarify why? I would like to fix content checks to be more thorough later, looking for obstacles.

Reply:
linux-vhd-content-test.sh runs through components.json today. But it's doing a fileSize check https://github.com/Azure/AgentBaker/blob/master/vhdbuilder/packer/test/linux-vhd-content-test.sh#L43

Rather than a 1:1 of the binaries. 

This change I've just made actually correctly failed this test because the swift and overlay cni's where added with this unpack check, but I didn't adjust the components.json to look for a non *.tgz